qemu/tests/virtio_port_hotplug.py

```diff
@@ -28,35 +29,51 @@ def run(test, params, env):
     :param params: Dictionary with the test parameters.
     :param env:    Dictionary with test environment.
     """
-
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
+    os_type = params["os_type"]
     timeout = int(params.get("login_timeout", 360))
-    if params["os_type"] == "windows":
-        run_bg_test(test, params, vm)
-    for repeat in range(int(params.get("repeat_times", 1))):
-        repeat += 1
+    module = params.get("modprobe_module")
+    check_module = params.get("check_module", "yes")
+    session = vm.wait_for_login()
+    if os_type == "windows":
+        driver_name = params["driver_name"]
+        session = utils_test.qemu.windrv_check_running_verifier(
+            session, vm, test, driver_name)
+        time_sleep = 0.5
+    else:
+        time_sleep = 2
+        params['file_size'] = 2000
```

Comment:
Incorrect parameter name,  it is `filesize`.  
It can be defined in cfg,  for this case, i see the setting is `filesize = 100`

Reply:
update and thanks !